src/main/java/com/redhat/darcy/webdriver/internal/WebDriverTarget.java

```diff
@@ -31,5 +31,9 @@
  * @see WebDriverTargets
  */
 public interface WebDriverTarget {
+    /**
+     * @throws org.openqa.selenium.NotFoundException if this target is not able to be switched to,
```

Comment:
Not seeing where this throws the said exception anywhere? I do see implementations throwing `NoSuchWindowException` in places though. Just wondering!